academy/4-my-own-chain/stored-game.md

```diff
@@ -289,7 +289,95 @@ Your stored game stores are only strings. But you know that they represent `sdk.
     )
```

Comment:
I feel like the step 3 should be done as step 1. Now we are using the errors before defining them, with in the end does not matter, but it feels _weird_ to me.